.travis.yml

```diff
@@ -36,6 +36,9 @@ matrix:
       env: STABILITY=beta
     - php: 7.1
       env: TARGET=csfixer_dry_run
+    - php: 7.2
+      env: STABILITY=beta SYMFONY_VERSION=4.1.*
```

Comment:
There is no need for this entry as Symfony 4.1 stable has been released. Travis automatically checks against the latest stable Symfony version.